src/errors/ForbiddenError.ts

```diff
@@ -0,0 +1,13 @@
+// Import base error
+import { BaseIsomerError } from "@errors/BaseError"
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Hmm if we feel as a team this is not impt eslint rule, could we change this at the global settings at a separate pr?

Reply:
shure